source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -1,167 +1,194 @@
+The following examples use:
 
-#. Get a listing of your newly-created bundle.
+* A stock installation of |CL| with all `Prerequisites`_.
 
-   .. code-block:: console
+Example 1: Add custom helloclear bundle
+=======================================
+
+This example shows the basic steps of adding a custom bundle from a local repo.
 
-      $ sudo swupd bundle-list -a
+#. Check that `helloclear` does not exist on your system:
 
-   The listing includes all upstream bundles.
+   .. code-block:: bash
 
-#. Add your bundle.
+      helloclear
 
    .. code-block:: console
 
-      $ sudo swupd bundle-add [bundle-name]
+      helloclear: command not found
 
-.. note:: 
+#. Follow "Option 1: Build a new RPM" from :ref:`autospec` to create a new
+   `helloclear` RPM.
 
-   You can also update your system to the latest upstream version using
-   this command:   
-   
-   .. code-block:: console
+   The resulting RPMs are in `~/clearlinux/packages/helloclear/rpms`.
+
+#. Create a new repo.
+
+   #. Create a local repo folder and copy the new `helloclear` RPM files into
+   the repo:
+
+      .. code-block:: bash
+
+         mkdir ~/mixin-repo
+         cp ~/clearlinux/packages/helloclear/rpms/helloclear-v1.0-1.x86_64.rpm ~/mixin-repo
+         cp ~/clearlinux/packages/helloclear/rpms/helloclear-bin-v1.0-1.x86_64.rpm ~/mixin-repo
+
+   #. Create the repo data:
+
+      .. code-block:: bash
+
+         cd ~/mixin-repo
+         createrepo_c .
+
+   #. Add the repo name, making sure to use your username for the correct path:
+
+      .. code-block:: bash
 
-      $ sudo swupd update
+         sudo mixin repo add mylocalrepo file:///home/<usrname>/mixin-repo/
 
-Optional: Revert your system back to 100% upstream
-**************************************************
+#. Create custom bundle with the new `helloclear` RPM. Add `helloclear` to the
+   `helloclear-bundle` bundle and build the bundle:
+
+      .. code-block:: bash
+
+         sudo mixin package add helloclear --bundle helloclear-bundle
+         sudo mixin build
+
+#. Migrate your |CL| to your custom mix. Check your version before and after the
+   update, to see the switch to your custom mix:
+
+   .. code-block:: bash
+
+      sudo swupd check-update
+      sudo swupd update --migrate
+      sudo swupd check-update
+
+#. Install your custom bundle. Check that the `helloclear-bundle` is now
+   available and install it to your system:
+
+   .. code-block:: bash
+
+      sudo swupd bundle-list -a | grep helloclear-bundle
+      sudo swupd bundle-add helloclear-bundle
+
+#. Test for `helloclear` again to see that it is installed:
+
+   .. code-block:: bash
+
+      helloclear
+
+#. Revert your system back to upstream (optional):
+
+   .. code-block:: console
 
-If you want to revert your |CL| system back to the official upstream
-version, use this command:
+      sudo swupd verify --fix --picky --force -m <upstream-version-number> -C /usr/share/clear/update-ca/Swupd_Root.pem
```

Comment:
There appears to be inconsistencies on whether to use [] pr <> for substitutions.  Can we sort this out?  In the past, I was told to use [].

Reply:
@bktan8 you are right that it's inconsistent across some of the docs. We've started a convo to better define how to mark substitutions, as well as how to refer to the users home dir. Using <> is correct for substitutions, per the contrib guide: https://clearlinux.org/documentation/clear-linux/reference/collaboration/structure-formatting#inline-text-formatting .  I'll add this to our general clean-up checklist.